features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/publisher/js/logic/lifecycle.asset.js

```diff
@@ -115,7 +115,7 @@ $(function(){
 
                 var out='<ul>';
 
-                var obj=JSON.parse(response);
+                var obj = response;
```

Comment:
removed redundant variable 'obj' in commit 35f4761eb2773e7e414629315583e0ca406df47d of PR #240